src/containers/layout/notifications-menu/NotificationsMenu.tsx

```diff
@@ -0,0 +1,81 @@
+import { FC } from 'react'
+import { Link as RouterLink } from 'react-router-dom'
+import { useTranslation } from 'react-i18next'
+
+import IconButton from '@mui/material/IconButton'
+import Typography from '@mui/material/Typography'
+import CloseRoundedIcon from '@mui/icons-material/CloseRounded'
+import Link from '@mui/material/Link'
+
+import AppMenu from '~/components/app-menu/AppMenu'
+import AppButton from '~/components/app-button/AppButton'
+import { ButtonVariantEnum, Notification, SizeEnum } from '~/types'
+import { styles } from '~/containers/layout/notifications-menu/NotificationsMenu.styles'
+import { liksByType } from '~/containers/layout/notifications-menu/NotificationsMenu.constants'
+
+interface NotificationsMenuuProps {
+  anchorEl: Element | null
+  onClose: () => void
+  items: Notification[]
+  onDelete: (item: Notification) => void
+  onClear: () => void
+}
+
+const NotificationsMenu: FC<NotificationsMenuuProps> = ({
+  anchorEl,
+  items,
+  onClear,
+  onDelete,
+  onClose
+}) => {
+  const { t } = useTranslation()
+
+  const handleLinkClick = (item: Notification) => {
+    onClose()
+    onDelete(item)
+  }
+
+  const menuItems = [
+    ...items.map((item) => (
+      <Typography key={item._id} sx={styles.menuItem}>
+        <Link
+          component={RouterLink}
+          onClick={() => handleLinkClick(item)}
+          sx={styles.link}
+          to={liksByType[item.type]}
+        >
+          {t(`header.notifications.messages.${item.type}`)}
+        </Link>
+        <IconButton onClick={() => onDelete(item)}>
+          <CloseRoundedIcon color='primary' fontSize={SizeEnum.Small} />
```

Comment:
i think it would good to create enum for color or move this two properties of styles in `NotificationsMenu.styles.ts` and use here in sx

Reply:
fixed